### ISSUE TITLE
Reproduce Results on Pokemon

### ISSUE THREAD

**Issue by Zhendong-Wang** [autonomousvision/projected_gan#66]
Hi, I am trying to run experiments with the projected GAN model. I randomly chose one dataset, pokemon (my love), to run. I used the exact same command as the instruction code, only except using 4 gpus, showing below. 

python train.py --outdir=./training-runs/ --cfg=fastgan --data=./data/pokemon256.zip --gpus=4 --batch=64 --mirror=1 --snap=50 --batch-gpu=8 --kimg=1000

It reports in the paper that the FID score is 26.36 at 0.8 M images. I got 37.47 instead. 
<img width="987" alt="Screen Shot 2022-03-04 at 18 24 31" src="https://user-images.githubusercontent.com/43709979/156859056-4f45e913-d6f7-4633-82e6-e2e1270a6697.png">

Wondering what command should I use to reproduce the results? Also, found in the supplementary it says the learning rate is set to 2e-4, while in the code, the default will set to 2e-3, I don't know which one is the correct one. 

Thanks!

**Comment by woctezuma** [comment]
Related:
- https://github.com/autonomousvision/projected_gan/issues/26

**Comment by Zhendong-Wang** [comment]
Thanks for the response! I will try the `--cfg fastgan_lite` setting, and get back to you.

**Comment by Zhendong-Wang** [comment]
Thanks a lot! by usiing  `--cfg fastgan_lite `, now I can have FID 26.44 on pokemon, which matches the results from paper. Just it takes more images for training, like 5.0 M imgs, but I will ignore this.

**Comment by xl-sr** [comment]
The longer training might be due to the hypersphere mapping I added (first line of the generator forward pass). You could try without it to get the original speed, although I think the results are better visually with the mapping :)

**Comment by woctezuma** [comment]
I think you are correct, and that is what Axel meant.

https://github.com/autonomousvision/projected_gan/blob/8480bdd9150a44c31874ffa69835e10a4f6009c4/pg_modules/networks_fastgan.py#L58-L60

**Comment by xl-sr** [comment]
huh, yeah, that's correct. I'd expected it to improve in speed again... I'll investigate, thanks for checking! :)

**Comment by Torment123** [comment]
@Zhendong-Wang Hi, I met the same problem, however, after I set --cfg=fastgan_lite, and using the exact same command python train.py --outdir=./training-runs/ --cfg=fastgan_lite --data=./data/pokemon256.zip --gpus=4 --batch=64 --mirror=1 --snap=50 --batch-gpu=8 --kimg=1000, the best FID I got after training is 31.80,  did you set the total training time as 1000kimg, or longer to get the 26.44 result? also, did you use a learning rate of 2e-4 instead of 2e-3? thanks

**Comment by Zhendong-Wang** [comment]
@Torment123 I trained it longer for like --kimg=10000, and the learning rate is actually fixed in the code, which doesn't need tuning.

**Comment by Torment123** [comment]
Hi, thank you for the reply
